packages/packagers/js/src/helpers.js

```diff
@@ -29,6 +31,9 @@ if (parcelRequire == null) {
   };
 
   $parcel$global[${JSON.stringify(parcelRequireName)}] = parcelRequire;
+  parcelRegister = parcelRequire.register;
```

Comment:
Is it enough to just have `var parcelRegister = parcelRequire.register;` after the `if (parcelRequire == null)` block? Why do you need to split the declaration and assignment?

Reply:
I think in the case where `$parcel$global[${JSON.stringify(parcelRequireName)}];` is undefined.
```javascript
var parcelRequire = $parcel$global[${JSON.stringify(parcelRequireName)}]; // undefined
var parcelRegister = parcelRequire.register; // Cannot read properties of undefined
```

Reply:
Good point. Updated, please review

Reply:
I guess the `var` gets hoisted to the top of the function, but I think just to be clearer as to what is happening the code should look like:
```js
var parcelRequire = ....
var parcelRegister;
if (parcelRequire == null) {
   // ...
}
parcelRegister = parcelRequire.register;
```